docs/local/hosted.md

```diff
@@ -1,31 +1,34 @@
-# Changing from local to hosted CodeSee
+# Changing from CodeSee Local to hosted CodeSee
```

Comment:
What do you think about also renaming it to CodeSee Hosted, to be consistent with CodeSee Local?

Reply:
Sounds good to me! Does this need wider consultation - we're effectively picking the brand names of the product.

Reply:
Good question! @jleven What do you think?

Reply:
I believe this is already set (please correct me if I'm wrong @jleven), so please go for it!

Reply:
Done! Also added some comments in the install guides to clarify hosted: true. Ready for re-review